Functions.Templates/Resources/Resources.resx

```diff
@@ -1876,4 +1876,10 @@ function app.</value>
   <data name="kafka_consumergroup_label" xml:space="preserve">
     <value>Consumer Group</value>
   </data>
+  <data name="SqlInputBinding_description" xml:space="preserve">
+    <value>Takes a SQL query to run and returns the output of the query.</value>

```

Comment:
I feel like we should have the overall description mention that this will create an HTTP trigger, thoughts @dzsquared ?

Reply:
ping @dzsquared

Reply:
A C# function that will be run on an HTTP trigger and returns the result of a provided SQL query.

Reply:
"A C# function that will be run on an HTTP trigger and takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated)."
Does this sound okay for the output binding description? @dzsquared

Reply:
yep, that output binding description looks good